src/physics/cam/clubb_intr.F90

```diff
@@ -4150,6 +5477,9 @@ subroutine clubb_tend_cam( state,   ptend_all,   pbuf,     hdtime, &
       !  Use correct qflux from cam_in, not lhf/latvap as was done previously
       te_b = te_b + (cam_in%shf(i)+cam_in%cflx(i,1)*(latvap+latice)) * hdtime
 
+      ! subtract enthalpy of falling precip from tb
+      te_b = te_b - prec_sh(i)*1000._r8*latice*hdtime
+
```

Comment:
hi-lighting new code for you to look over and make sure that is meant to apply generally and not for clubbmf specifically.

Reply:
I think this is fine since the ```prec_sh``` pbuf is initialized to zero in ```cam/convect_shallow.F90``` for CAM6 and ```cam7/convect_diagnostics.F90``` for CAM7, when CLUBB is turned on. Only when running CLUBB+MF will it get set to the precip in the MF plumes (in clubb_intr).